src/options/modules/CustomOptionTriggers.js

```diff
@@ -181,6 +183,20 @@ function adjustPickerResultTypeOption(param) {
     return AutomaticSettings.Trigger.overrideContinue(param.optionValue);
 }
 
+/**
+ * Adjusts the emojiSearch->maximumResults setting for loading.
+ *
+ * @private
+ * @param {object} emojiSearch The emojiSearch option group object
+ * @returns {Promise}
+ */
+function adjustEmojiSearchMaximumResultsOnLoad(emojiSearch) {
```

Comment:
Unfortunately, this does not work yet. Apparently none of the `addCustomLoadOverride` are called, but one would have noticed it before? Is this a new bug hmm?

Here a video showing the bug (the aim is that the slider goes to the end aka 21, currently, aka to the right. It does not, which indicates it is initialised with an invalid/null etc. value.)